src/tests/ftest/pool/evict_test.py

```diff
@@ -266,7 +266,7 @@ def test_evict_bad_server_name(self):
         """
         Test evicting a pool using an invalid server group name.
 
-        :avocado: tags=all,pool,pr,full_regression,small,poolevict
+        :avocado: tags=all,pool,full_regression,small,poolevict
```

Comment:
It's a little confusing since the previous state was to run in both PRs and the weekly test run, which we generally don't do -- it's one or the other, so do we intend to just run this weekly instead of on every PR from now on?

Or is this just a temporary move to weekly while we fix the issue?  If so, what triggers it moving back to `pr` once it's fixed?  Will the developer fixing it know that it should be moved back?

Reply:
I left a comment in the bug ticket to re-enable test in PR when issue is fixed
https://jira.hpdd.intel.com/browse/DAOS-4410

Reply:
Great!  That's exactly the sort of thing that is needed.